File: python/1.py
def sum_divisor_of_three_or_five(n):
  sum = 0
  
  for n in range(1, n):
    if n % 3 == 0 or n % 5 == 0:
      sum += n

  return sum

print sum_divisor_of_three_or_five(10)
print sum_divisor_of_three_or_five(1000)

File: rust/1.rs
fn main() {
    println!("{}", sum_divisor_of_three_or_five(10));
    println!("{}", sum_divisor_of_three_or_five(1000));
}

fn sum_divisor_of_three_or_five(n: u32) -> u32 {
    let mut sum = 0;

    for i in 1..n {
        if i % 3 == 0 || i % 5 == 0 {
            sum += i;
        }
    }

    return sum;
}
